cdk/__tests__/main-test.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) HashiCorp, Inc
+// SPDX-License-Identifier: MPL-2.0
+import 'cdktf/lib/testing/adapters/jest'; // Load types for expect matchers
+// import { Testing } from "cdktf";
+
+describe('My CDKTF Application', () => {
+  // The tests below are example tests, you can find more information at
+  // https://cdk.tf/testing
+  it.todo('should be tested');
```

Comment:
let's add some basic tests

Reply:
The tests here would only check if the resources are there in the local files not against the aws.
I'll have to read a bit about writing tests, I'll add some basic tests tomorrow.